deploy/eck-operator/values.yaml

```diff
@@ -236,3 +236,6 @@ global:
   # kubeVersion is the effective Kubernetes version we target when generating the all-in-one.yaml.
   kubeVersion: 1.21.0
 
+# When using AWS EKS and an altnerate CNI inconjunction with webhook enabled, hostNetwork needs to enabled to allow for
```

Comment:
```suggestion
# HostNetwork allows a Pod to use the Node network namespace.
# This is required to allow for communication with the kube API when using some alternate CNIs in conjunction with webhook enabled.
```

How about moving this just below the `webhook` field L123 where there is currently `softMultiTenancy`?

Reply:
Works for me, updated the field description and moved the hostNetwork boolean to the webhook section.